com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/CreateAppStep.java

```diff
@@ -233,4 +234,9 @@ private void bindService(CloudFoundryOperations client, String appName, String s
         return (bindingParameters == null) ? null : bindingParameters.get(serviceName);
     }
 
+    protected static Map<String, Object> getParametersForBinding(CloudServiceBinding cloudServiceBinding) {

```

Comment:
Could this be renamed to getBindingParametersOrDefault? or something similar?

Reply:
Done